### DISCUSSION TITLE
Please explain how to use custom sources - insufficient docs

### DISCUSSION THREAD

**Discussion by mklueh** [nuxt/content#1615] [Q&A]
Hello,

I was looking into the docs seeing it's possible to fetch data from GitHub, however I do not understand the API.

https://content.nuxtjs.org/api/configuration#sources

Iand added some questions I have to the config sample:

```
export default defineNuxtConfig({
  content: {
    sources: [
      'content',
      {
        name: 'fa-ir', //what impact does it have?
        prefix: '/fa',
        driver: 'fs', // fs - does it load the node fs module and use it? what else can be passed here? How can I make REST calls and fetch JSON?
        // ...driverOptions
        base: resolve(__dirname, 'content-fa') // what else can I do with that? when is it needed? How to customize fetching?
      }
    ]
  }
})
```

If it's possible to fetch from GitHub, then it should also be possible to fetch data from any CRM that supports REST, correct?

To sum it up, could anyone provide any more detailed example? Because I don't understand it

**Comment by ManasMadrecha** [comment]
Related to #1049
